lib/core/wireprotocol/command.js

```diff
@@ -17,6 +21,15 @@ function command(server, ns, cmd, options, callback) {
     return callback(new MongoError(`command ${JSON.stringify(cmd)} does not return a cursor`));
   }
 
+  if (!isClientEncryptionEnabled(server)) {
+    _command(server, ns, cmd, options, callback);
```

Comment:
Since we are doing this at the `command` level, will this break APM?

Reply:
I don't see why it would, do you have a thought here? APM is actually used to run the YAML tests, so I think we're accurately flexing that it still works as expected.